lib/swagger/generator.rb

```diff
@@ -18,6 +23,14 @@ module SwaggerGenerator
     # http://files.slatestudio.com/gG30
     ALLOW_TYPES  = %w(Object String Integer Array Date Mongoid::Boolean Symbol)
 
+    def swagger_options(options)
+      self.swagger_base_path           = options[:base_path]
+      self.swagger_collection_name     = options[:collection_name]
+      self.swagger_relative_path       = options[:relative_path]
+      self.ignore_custom_actions       = options[:ignore_custom_actions]
```

Comment:
I think this also should have `swagger_` prefix that illustrates that this class attribute belongs to to swagger module.